src/frame/getFrameHtmlResponse.ts

```diff
@@ -3,6 +3,7 @@ import { FrameMetadataType } from './types';
 type FrameMetadataHTMLResponse = FrameMetadataType & {
   ogDescription?: string;
   ogTitle?: string;
+  xmtpContent?: string;
```

Comment:
It's probably clearer to call this `xmtpMinSupportedVersion`

Reply:
I like that idea, just made the change. Thank you!